WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -397,3 +397,7 @@ table.admin-competitions {
     }
   }
 }
+
+.championship-trophy {
+  color: #e64503;
```

Comment:
Perhaps this should be either of https://github.com/thewca/worldcubeassociation.org/blob/793e9cd4a2e391a41593b7bb5761e43395049500/WcaOnRails/app/assets/stylesheets/variables.scss#L9-L10
If not, then feel free to add a new variable instead of hard-coding the color.

Reply:
I was going for the WCA orange that's used throughout the site, which I found was $link-color. I'll make that change.